plugins/docker_binary_builder/app/models/binary_builder.rb

```diff
@@ -30,6 +40,12 @@ def build
     @image.remove(force: true) if @image
   end
 
+  def run_pre_build_script
```

Comment:
Is there an alternative way to hook onto pre-docker binary build phase?

the docker binary build seems to be triggered at start of job execution and hooking on to deploy:build in Capfile doesnt seems to be able to intercept it...